src/components/navigation/navigation/navigation.ts

```diff
@@ -142,13 +150,30 @@ export class SbbNavigationElement extends UpdateScheduler(LitElement) {
       return;
     }
     this._state = 'opening';
+    this._setActiveAction();
     this.startUpdate();
 
     // Disable scrolling for content below the navigation
     this._scrollHandler.disableScroll();
     this._triggerElement?.setAttribute('aria-expanded', 'true');
   }
 
+  private _setActiveAction(): void {
+    const activeActions = Array.from(
```

Comment:
It is very well possible, that I don't understand what is happening here. 😅 
Only the first element with .sbb-active should be active and if it has a connected section, that section should be opened.
The forEach throws me off here. But as mentioned, this could just be me.